cmd/internal/cli/actions.go

```diff
@@ -48,7 +48,7 @@ func getCacheHandle(cfg cache.Config) *cache.Handle {
 func actionPreRun(cmd *cobra.Command, args []string) {
 	// For compatibility - we still set USER_PATH so it will be visible in the
 	// container, and can be used there if needed. USER_PATH is not used by
-	// singularity itself in 3.9+
+	// apptainer itself in 3.9+
```

Comment:
Should be 1.0+

Reply:
@dwd please write an issue for deferral and mark this resolved

Reply:
Defer to #47